lib/marked.js

```diff
@@ -1084,13 +1084,33 @@ Parser.prototype.tok = function() {
  * Helpers
  */
 
+var escapeTest = /[&<>"']/;
```

Comment:
These should be declared inside escape() IMO

Reply:
No, marked do not have to recreate the same `RegExp` instance every time. This reduces performance and increases memory usage.

Reply:
Ok, I made them static.